File: main/src/main/java/com/sedmelluq/discord/lavaplayer/filter/PcmFormat.java
package com.sedmelluq.discord.lavaplayer.filter;

/**
 * Describes the properties of PCM data.
 */
public class PcmFormat {
  /**
   * Number of channels.
   */
  public final int channelCount;
  /**
   * Sample rate (frequency).
   */
  public final int sampleRate;

  /**
   * @param channelCount See {@link #channelCount}.
   * @param sampleRate See {@link #sampleRate}.
   */
  public PcmFormat(int channelCount, int sampleRate) {
    this.channelCount = channelCount;
    this.sampleRate = sampleRate;
  }
}

File: main/src/main/java/lavaplayer/filter/PcmFormat.kt
package lavaplayer.filter

/**
 * Describes the properties of PCM data.
 *
 * @param channelCount See [.channelCount].
 * @param sampleRate   See [.sampleRate].
 */
data class PcmFormat(
    /**
     * Number of channels.
     */
    @JvmField val channelCount: Int,
    /**
     * Sample rate (frequency).
     */
    @JvmField val sampleRate: Int
)
